isatools/isatab_full.py

```diff
@@ -0,0 +1,297 @@
+
+from __future__ import absolute_import
+
+import os
+
+import numpy as np
+import pandas as pd
+
+from isatools.isatab_meta import AbstractParser, InvestigationParser, TableParser
```

Comment:
I considered putting `TableParser` in this module, rather than `isatab_meta`.  However, that wasn't straightforward since the `InvestigationParser` depends on it at the moment by using some constants defined in `TableParser` (thus moving it would introduce a circular dependency).

Reply:
Should be OK as long as it doesn't cause any errors at runtime to use the `InvestigationParser`.